tmpfiles.d/journal-nocow.conf

```diff
@@ -0,0 +1,27 @@
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+# See tmpfiles.d(5) for details
+
+# Set the NOCOW attribute for directories of journal files. This flag
+# is inheredited by their new files and sub-directories. Matters only
```

Comment:
s/inherideted/inherited/

Reply:
https://github.com/systemd/systemd/pull/6804 The same typo is also in upstream. Let's fix that after the patch is merged.